app/components/Capture.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Button } from 'react-bootstrap';
```

Comment:
react-bootstrap is being used only for the grid. Only selected grid css is imported. Don't use this button.

Reply:
Removed